app/src/integrationTests/java/com/example/networking/RestServiceIntegrationTest.java

```diff
@@ -38,7 +38,7 @@ public void afterEveryTest() throws IOException {
     public void shot_request_test_network_layer() throws IOException {
         mockWebServerHelper.enqueueJsonResponseFromFileForMockWebServer(mockWebServer, "shots_response.json");
 
-        List<ShotResponse> shotResponseList = restService.getShots().toBlocking().value();
+        List<ShotResponse> shotResponseList = restService.getShots().toObservable().blockingSingle();
```

Comment:
I don't quite understand what you are talking about. What do you mean by continuous stream? This is just a Single<List<>>